ci-operator/config/openshift/metallb/openshift-metallb-release-4.10.yaml

```diff
@@ -50,25 +50,31 @@ tests:
   steps:
     cluster_profile: aws
     workflow: openshift-e2e-aws
-- as: metallb-l2-e2e-metal-ipi
+- as: metallb-e2e-metal-ipi
   steps:
     cluster_profile: packet
     env:
       DEVSCRIPTS_CONFIG: |
         IP_STACK=v4
         NETWORK_TYPE=OpenShiftSDN
-        ENABLE_METALLB_MODE=l2
+        ENABLE_METALLB=true
         METALLB_REPO=https://github.com/openshift/metallb.git
+        METALLB_OPERATOR_REPO=https://github.com/openshift/metallb-operator.git
+        METALLB_OPERATOR_COMMIT=22afa1a
```

Comment:
Same as above. In this case, can't it be set to "release-4.10" to follow the OCP 4.10 release branch?